### PR TITLE
feat: add info about TopicScopes

### PR DESCRIPTION
**Pull request by cprice404**
This commit adds some initial information about TopicScopes for
auth tokens.

Also updates the web SDK page to use injected code samples for
auth/refresh, and adds some injected example code for refresh
on the auth tokens page.

### CODE REVIEW

File: docs/develop/sdks/web/index.md [hunk 1]
```diff
@@ -48,28 +53,16 @@ Here's an example import statement for the web SDK:
...
+To create a Momento auth token for use in the browser, you will generally have a web application using another Momento SDK such as the node.js SDK. First, you will need to instantiate a Momento `AuthClient`:
+
+<SdkExampleCodeBlock language={'javascript'} snippetId={'API_InstantiateAuthClient'} />
+
+Then you will use the `generateAuthToken` API to create a token that you can vend to the browser:
+
+<SdkExampleCodeBlock language={'javascript'} snippetId={'API_GenerateAuthToken'} />
+
+For more information on Momento auth tokens, including `TokenScope` for authorization, and how to refresh expiring tokens,
+see [Working with Momento auth tokens](/develop/guides/working-with-momento-auth-tokens).
```
Comment (NoSQLKnowHow) [inline docs/develop/sdks/web/index.md]: remove carriage return, pls.

File: docs/develop/sdks/web/index.md [hunk 2]
```diff
@@ -48,28 +53,16 @@ Here's an example import statement for the web SDK:
...
 
 ## FAQ
```
Comment (NoSQLKnowHow) [inline docs/develop/sdks/web/index.md:66]: If there are no questions, this section should be deleted.

Comment (cprice404) [reply docs/develop/sdks/web/index.md:66]: lol I didn't add this section, it has your question in it :)

File: docs/develop/sdks/web/index.md (hunk 1 above)
Comment (cprice404) [reply docs/develop/sdks/web/index.md]: you have an eagle eye for this :)

Comment (cprice404) [reply docs/develop/sdks/web/index.md]: fixed in https://github.com/momentohq/public-dev-docs/pull/240/commits/5a8c31ee77690d25960fd48e682780ec84ee7449

**Review by NoSQLKnowHow — APPROVED** [review]

**Pull request merged**